docs/tempo/website/getting-started/quickstart-tempo.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Quickstart with Tempo
+---
+
+If you are running an application locally that is already instrumented with tracing,
+this guide will help set up the components required to send & query traces in Tempo.
+
+## Step 1: Spin up Tempo backend
+
+First, set up a docker network as shown -
+
+```
+docker network create docker-tempo
+```
+
+Next, download the [configuration file](https://github.com/grafana/tempo/blob/master/example/docker-compose/etc/tempo-local.yaml) using the following command -
+
+```
+curl -o tempo-local.yaml https://raw.githubusercontent.com/grafana/tempo/master/example/docker-compose/etc/tempo-local.yaml
+```
+
+The config file above configures Tempo to listen on default ports for a number of protocols.
+List of protocols and their default ports:
+
+|  Protocol    |   Port  |
+|  ---         |   ---   |
+|  OpenTelemetry  | 55680 |  # Grafana Agent uses this.
+|  Jaeger - Thrift Compact | 6831 |  # Jaeger Golang client uses this when used with JAEGER_AGENT_HOST & JAEGER_AGENT_PORT
+|  Jaeger - Thrift Binary |  6832  |
+|  Jaeger - Thrift HTTP |  14268 |  # Jaeger Golang client uses this when used with JAEGER_ENDPOINT
+|  Jaeger - GRPC |  14250  | # Jaeger Agent uses this.
+|  Zipkin  | 9411 |
+
+Choose the port corresponding to the protocol you wish to use to send traces to Tempo. For this example we have used Jaeger - Thrift
+Compact format (port 6831).
+
+```
+docker run -d --rm -p 6831:6831/udp --name tempo -v $(pwd)/tempo-local.yaml:/etc/tempo-local.yaml \
+    --network docker-tempo \
+    grafana/tempo:latest \
+    -config.file=/etc/tempo-local.yaml
+```
+
+## Step 2: Spin up Grafana for visualizing traces
+
+```
+docker run -d --rm -p 3000:3000 \
+    --network docker-tempo \
+    grafana/grafana:7.5.1
+```
+
+Make sure the UI is accessible at http://localhost:3000.
```

Comment:
If I'm a new user testing Tempo, and I've never touched Grafana, seeing the login form is strange. 

Maybe it's interesting to add here the default user and password.

Reply:
> I've never touched Grafana

😱  

haha, you're right, I just removed auth in the example.